### PR TITLE
Add min and max version of OS for policies

### LINKED ISSUE #443: Min and max  version of mobile OS for each policy
Hi, @flyve-mdm/frontend-development

- Delete is_android_policy and is_apple_molicy fields
- Create new fields on glpi_plugin_flyvemdm_policies
  - android_min_version
  - android_max_version
  - apple_min_version
  - apple_max_version

If android_mix_version == android_max_version == 0 then this means the policy is not available for this operating system

If apple_mix_version == apple_max_version == 0 then this means the policy is not available for this operating system

Extra task related to https://github.com/flyve-mdm/android-mdm-agent/issues/373

we need to implement a generic task status feedback value (the string "not supported" may be a good candidate)

### PR DESCRIPTION
**Pull request by Naylin15**
### Changes description
Add min and max version for policy according the operating system

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [x] Docs have been added/updated

### References

Closes #443
related: https://github.com/flyve-mdm/web-mdm-dashboard/issues/740

### CODE REVIEW

**Comment by Naylin15** [comment]
Hi @btry 

I added the changes you commented, also added the minimum versions according to the policy for Android since I already investigated this in [another issue](https://github.com/flyve-mdm/android-mdm-agent/issues/365), but I have to search for two policies I didn't have at the moment

**Comment by Naylin15** [comment]
Hi,

I added the min android version for display in the information, also fixed some typos in the policy's names - messages

**Comment by Naylin15** [comment]
Hi, @btry 

I made the changes requested

File: install/policies/deployment.php [hunk 1]
```diff
@@ -93,13 +99,15 @@
       'type_data'                           => '',
       'unicity'                             => 0,
       'plugin_flyvemdm_policycategories_id' => $category,
-      'comment'                             => __('Uninstall a file on the device',
+      'comment'                             => __('Uninstall a file on the device. Available since Android 4.2',
```
Comment (btry) [inline install/policies/deployment.php]: Please drop Available from **** in all policies, as I suggested in my previous intermediate review.

**Comment by btry** [comment]
@rafaelje this PR may be interesting for you, to get the mobile OS requirements for each policy.

**Comment by btry** [comment]
@Naylin15 I put the PR as WIP because these new data need development anyway in the dashboard. We need to merge when the dashboard is ready.

**Comment by Naylin15** [comment]
All right 👍

Commits squashed

**Comment by hectorerb** [comment]
Hi @btry what additional feature do you need on dashboard?

**Comment by btry** [comment]
@hector

When the admin wants to apply policies, he may need help. We need to provide documentation about the policies:
The comment field may contain several paragraphs of text. We need to show it (the current design of the dashboard allows us only something similar to a tooltip).

We also need to tell the availability on all mobile platforms: min and max version per operating system. Again this is to help the adnin in his decisions. We can show it at your convenience, prepended in the comment may be nice. (Other ideas welcome)

**Pull request merged**